tasks.md

```
@@ -0,0 +1,13 @@
+# Task Plan
+
+- Task 1: Update `useNormalizationSourceLines.ts` to include `overrideNormalizationSourceLine` mutation.
+- Task 2: Update `NormalizePage` in `page.tsx` to collect reason/note/data and call the new mutation.
+
+## Task 1: Update Hook
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)